python/pyarrow/tests/test_compute.py

```diff
@@ -1848,6 +1849,64 @@ def test_assume_timezone():
     result.equals(pa.array(expected))
 
 
+units = (
+    "nanosecond",
+    "microsecond",
+    "millisecond",
+    "second",
+    "minute",
+    "hour",
+    "day",
+)
+unit_shorthand = {
+    "nanosecond": "ns",
+    "microsecond": "us",
+    "millisecond": "L",
+    "second": "s",
+    "minute": "min",
+    "hour": "H",
+    "day": "D"
+}
+
+
+# TODO: We should test on windows once ARROW-13168 is resolved.
+@pytest.mark.pandas
+@pytest.mark.skipif(sys.platform == 'win32',
+                    reason="Timezone database is not available on Windows yet")
+@pytest.mark.parametrize('unit', units)
+def test_round_temporal(unit):
+    # timezones = ["UTC", "US/Central", "Asia/Kolkata",
+    #              "Etc/GMT-4", "Etc/GMT+4", "Australia/Broken_Hill"]
+    # timezones = ["UTC", "Etc/GMT+4"]
```

Comment:
Are these lines meant to be commented out still? (And below)

Reply:
They are commented out to please the CI. Still working on the C++ logic.